packages/client/src/views/PrintCertificate/usePrintableCertificate.ts

```diff
@@ -196,10 +196,20 @@ export const usePrintableCertificate = (declarationId: string) => {
     dispatch(goToHomeTab(WORKQUEUE_TABS.readyToPrint))
   }
 
-  const handleEdit = () =>
+  const handleEdit = () => {
+    // Delete certificate properties during print record corrections
+    // since correction flow doesn't handle certificates
+    if (declaration?.data?.registration.certificates) {
+      delete (declaration?.data?.registration as { certificates: any })
+        ?.certificates
+      dispatch(modifyDeclaration(declaration))
+      dispatch(writeDeclaration(declaration))
```

Comment:
We should maintain immutability when updating the declaration. Although it probably won't affect anything now, might save us from tearing our hair out due to some bug in the future. :wink: